scripts/question_data.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO online_hearing(online_hearing_id, external_ref, jurisdiction_id) VALUES ('d9248584-4aa5-4cb0-aba6-d2633ad5a375', 'e2e_test_oh', 1);
```

Comment:
Okay, I see why this is here. Once the online hearing code is working properly this won't be necessary. Plus we don't want to hard code test data, unless we also remove it in the AnswerStep. Also, I *think*, you can achieve the same thing by calling the @Sql annotation to inject. See https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/context/jdbc/Sql.html